recipes/dirty_cat/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "dirty_cat" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dirty_cat-{{ version }}.tar.gz
+  sha256: 2b44f38462ad4e836861fd7a86caccd49d3490e47a2617c12edebb3c6a53dc16
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - joblib
+    - numpy >=1.16
+    - pandas >=1.1.5
+    - python >=3.6
+    - requests
+    - scikit-learn >=0.21
+    - scipy >=1.2
+
+test:
+  imports:
+    - benchmarks
```

Comment:
This means the package is installing its benchmarks as a globally importable `benchmarks` package--which could quite easily conflict, and doesn't seem useful. Can you remove the source before install or hack the `setup.py`?

Reply:
I'll give it a try with a patch